lib_collections/models.py

```diff
@@ -629,25 +1320,26 @@ def viewer(self, request, *args, **kwargs):
 
     # panels within the 'Collection' tab in the admin interface
     collection_panels = [
-        FieldPanel('digital_collection'),
-        MultiFieldPanel(
-            [
-                ImageChooserPanel('banner_image'),
-                ImageChooserPanel('banner_feature'),
-                FieldPanel('banner_title'),
-                FieldPanel('banner_subtitle'),
-            ],
-            heading='Banner'
-        ),
-        MultiFieldPanel(
-            [
-                FieldPanel('branding_color'),
-                FieldPanel('google_font_link'),
-                FieldPanel('font_family'),
-            ],
-            heading='Branding'
-        ),
+        # FieldPanel('digital_collection'),
+        # MultiFieldPanel(
+        #     [
+        #         ImageChooserPanel('banner_image'),
+        #         ImageChooserPanel('banner_feature'),
+        #         FieldPanel('banner_title'),
+        #         FieldPanel('banner_subtitle'),
+        #     ],
+        #     heading='Banner'
+        # ),
+        # MultiFieldPanel(
+        #     [
+        #         FieldPanel('branding_color'),
+        #         FieldPanel('google_font_link'),
+        #         FieldPanel('font_family'),
+        #     ],
+        #     heading='Branding'
+        # ),
```

Comment:
Yes.  They were originally carried over from the Exhibits admin panels, and then Kathy wondered whether we really needed them for digital collections.  So as sort of a compromise measure, we left the earlier code that defines this part of the `CollectionPage` model alone (i.e. no migrations necessary), but I commented out the code that creates the actual panels.

Reply:
Do you think it would be better to straight up delete this part?  My understanding was that we were basically undecided about whether to bring it back some day or completely delete it and migrate it out of the db.